tox.ini

```diff
@@ -16,9 +16,15 @@ commands =
 basepython = python3.8
 deps =
     {[testenv]deps}
-    Scrapy==2.7.0
-    url-matcher==0.5.0
-    w3lib==1.22.0
+    Scrapy>=2.11.0
+    url-matcher>=0.5.0
+    w3lib>=2.2.0
```

Comment:
It's a `pinned` env so these should be pinned to the minimal version (I guess it's just a copy-paste mistake).

Reply:
I needed to bump Scrapy to `2.11.0` since I get errors like `ImportError: cannot import name 'RequestFingerprinterProtocol' from 'scrapy.utils.request'`. It seems it wasn't available before this version.

On the other hand, `w3lib` was needed to be bumped since I get errors due to trailing `#`:

```
FAILED tests/test_processors.py::test_query_removal_except[args6-http://foo.example?foo=1&bar-http://foo.example?foo=1] - AssertionError: assert 'http://foo.example?foo=1#' == 'http://foo.example?foo=1'
FAILED tests/test_processors.py::test_query_removal_except[args7-http://foo.example?foo=1&bar-http://foo.example?foo=1&bar] - AssertionError: assert 'http://foo.e...le?foo=1&bar#' == 'http://foo.example?foo=1&bar'
FAILED tests/test_url_canonicalizer.py::test_url_canonicalizer_load - AssertionError: assert 'http://foo.e...?foo=1&bbn=1#' == 'http://foo.e.../?foo=1&bbn=1'
FAILED tests/test_url_canonicalizer.py::test_url_canonicalizer_multiple_rules_same_processor[1-1] - AssertionError: assert 'https://example.com#' == 'https://example.com'
FAILED tests/test_url_canonicalizer.py::test_url_canonicalizer_multiple_rules_same_processor[1-2] - AssertionError: assert 'https://example.com#' == 'https://example.com'
FAILED tests/test_url_canonicalizer.py::test_url_canonicalizer_multiple_rules_same_processor[2-1] - AssertionError: assert 'https://example.com#' == 'https://example.com'
```

But it would seem the min version to fix this would be `w3lib==2.0.1`. I'll update this one.